src/pages/address/Address.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from "react";
+import "./Address.css";
+import axios from "axios";
+import { AddressField } from "./components/address_field/AddressField";
+import { Modal } from "./components/modal/Modal";
+import SavedAddress from "./components/saved_address/SavedAddress";
+import { useAddress } from "./context/address-context";
+import { getUserToken } from "../../utils/TokenHelper";
+
+function Address() {
+  const [isAddingAddress, setIsAddingAddress] = useState(false);
+  const [isModalActive, setIsModalActive] = useState(false);
+  const [isEditActive, setIsEditActive] = useState(false);
+  const [addressId, setAddressId] = useState("");
```

Comment:
if the no. of state increase to 3 or more delegate into a reducer

Reply:
Ok will try that